.github/CODEOWNERS

```diff
@@ -24,11 +24,11 @@
 /specification/logs/ @tigrannajaryan
 
 # JVM semantic conventions approvers
-/semantic_conventions/metrics/process-runtime-jvm-metrics.yaml @trask
-/semantic_conventions/metrics/process-runtime-jvm-metrics-experimental.yaml @trask
+/semantic_conventions/metrics/process-runtime-jvm-metrics.yaml              @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-jvm-approvers
+/semantic_conventions/metrics/process-runtime-jvm-metrics-experimental.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-jvm-approvers
 
 # HTTP semantic conventions approvers
-/semantic_conventions/metrics/http.yaml @trask
-/semantic_conventions/trace/http.yaml @trask
+/semantic_conventions/metrics/http.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-http-approvers
```

Comment:
I think groups are fine since they can be edited in the UI without changing the owners file in the repository and act as a single source of truth. If we go with individual names then we might also have to reflect the changes in issue and PR templates as well as Github workflows once everything is set up nicely.